### DISCUSSION TITLE
-nset to create any wanted meshed sections

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#760] [Q&A] [answered]
Dear Romain,
First thank you so much for your useful software!

For example, if I use the T module to generate a cubic bar,
neper -T -n 1000 -domain "cube(100,100,1000)" -o bar

Then I want to define multiple surfaces (say 50 surfaces), they are measured surfaces in Abaqus (see the figure below which contains 3 surfaces). My question is, if I want to automatically generate these surface sections in M module, how could I play with the command -nset (or -faset) using the newest version of NEPER (4.6.2) ? And could this information be saved in .inp file?  To manually generate these surfaces in Abaqus could be really painfull!
![image](https://github.com/neperfepx/neper/assets/75380898/6d51889a-0005-4a55-97fd-404a338d37d5)

Thank you before!
Best regards

**Comment by rquey** [comment, marked as answer]
This looks more like a post-processing problem than a meshing problem. You could determine the nodes or elements about these locations using `-statnode x,y,z` and `-statelt x,y,z`, and go from there...

**Reply by damulaoshi** [reply]
Dear Romain,
Ok, thank you! I'll see how to do this in the post-process. Thanks again!
Best regards,
